Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -290,7 +290,7 @@
     <!-- XMSG: risk details - wash your hands, something like a bullet point -->
     <string name="risk_details_behavior_body_wash_hands">"Waschen Sie Ihre Hände regelmäßig."</string>
     <!-- XMSG: risk details - wear a face mask, something like a bullet point -->
-    <string name="risk_details_behavior_body_wear_mask">"Tragen Sie einen Mundschutz bei Begegnungen mit anderen Personen."</string>
+    <string name="risk_details_behavior_body_wear_mask">"Tragen Sie einen Mund-Nase-Schutz bei Begegnungen mit anderen Personen."</string>
```

Comment:
why not "Mund-Nasen-Schutz"?

Reply:
see also https://github.com/corona-warn-app/cwa-app-ios/pull/905

Line 84 of src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings :

https://github.com/corona-warn-app/cwa-app-ios/pull/905/files#diff-de816028e6169d535787f1bae78426b8L84-R84

Reply:
Thanks for the quick reply, changed.